extension/recording-a-log.md

```diff
@@ -14,42 +14,44 @@ Log recording is the core functionality provided by Fiddler Jam. A **Log** in Fi
 
-2. Expand the [**Capture Options**]({%slug fj-capture-options%}) and choose the preferred settings.
+1. (Optional) Expand the [**Advanced Options**]({%slug fj-capture-options%}) and choose the preferred settings. By default, all advanced options are turned on.
 
-3. Click **Start Capture** to begin recording. Fiddler Jam will start recording all network activity in your current browser tab.
+1. Click **Start Capture** to begin recording. Fiddler Jam will start recording all network activity in your current browser tab. Click anywhere in the browser tab to hide the Fiddler Jam window and to continue recording. To bring the Fiddler Jam window back, click the extension's icon.
 
-   ![Start capture screen with the capture options expanded](../images/ext/ext-images/extension-start-capturing-extended.png)
+   ![Start capture screen with the Advanced Options expanded](../images/ext/ext-images/extension-start-capturing-extended.png)
 
-   >important Chrome displays an informational banner: **"Fiddler Jam" started debugging this browser**. Closing the notification or clicking **Cancel** won't stop the capturing session. Use the **Stop Capture** button (see the 5th step) to cancel an ongoing recording.
+   >important Chrome displays an informational banner: **"Fiddler Jam" started debugging this browser**. Closing the notification or clicking **Cancel** won't stop the capturing session. Use the **Stop Capture** button (see the next step) to cancel an ongoing recording.
+   ![Chrome warning banner](../images/ext/ext-images/extension-chrome-notification.png)
 
-4. Click anywhere in the browser tab to hide the Fiddler Jam window and to continue recording. To bring the Fiddler Jam window back, click the extension's icon.
+1. Click  **Stop Capture** to end the recording.
 
-   At any time during recording, you can click **_Capturing…_** to see details about what Fiddler Jam records.
+   ![Stop capturing screen](../images/ext/ext-images/extension-stop-capturing.png)
 
-   ![Ongoing capturing screen](../images/ext/ext-images/extension-link-options-show-sessions.png)
+1. Before submitting the log, you could check the recorded log by clicking on **Capture Successful** line. The **Capture Details** screen will load with a list of the captured sessions, screenshots, console logs, etc. Go back to the previous screen to generate a link and submit the log to the cloud portal.
 
-5. Click  **Stop Capture** to end the recording.
+   ![Stop capturing screen](../images/ext/ext-images/extension-link-options-show-sessions.png)
 
-   ![Stop capturing screen](../images/ext/ext-images/extension-stop-capturing.png)
+1. At this point, the captured traffic is recorded as a log but is not yet saved anywhere. Choose the appropriate [**Sharing Options**]({%slug fj-link-options%}), use the **Get Link** button and proceed with [submitting the recorded log to the JAM portal]({%slug fj-submitting-logs%}#submit-via-link).
```

Comment:
`is not yet saved anywhere` sounds a bit confusing - is "saving" and "submitting / sharing" the same thing? What happens if I don't submit the log to the portal and close Fiddler Jam - is the log lost?

Reply:
Yep, your assumption is correct - not submitting the log will result in losing the recorded traffic.  I will provide extended description of the whole workflow and rethink using the whole **save** wording.